pocs/camera/camera.py

```diff
@@ -302,7 +302,13 @@ def take_observation(self, observation, headers=None, filename=None, **kwargs):
 
         # pop exptime from kwarg as its now in exptime
         exptime = kwargs.pop('exptime', observation.exptime.value)
-        exposure_event = self.take_exposure(seconds=exptime, filename=file_path, **kwargs)
+        
+        # Now move the filerwheel into position
+        if (self.filterwheel is not None) & (observation.filter_name is not None):
```

Comment:
```suggestion
        if (self.filterwheel is not None) and (observation.filter_name is not None):
```

Reply:
@lspitler  O.K. but why?

Reply:
Wrong operator. In Python `and` is the operator for combining `bool` values, whereas `&` is the operator for doing bitwise ANDing of binary numbers (bitmaps/bitmasks).